modules/client-configuration/pages/client-automating-preparation.adoc

```diff
@@ -0,0 +1,136 @@
+// later?  and again kernel options.
+
+Write, or browse and upload the ``Kickstart`` or ``AutoyaST`` profile.
+
+[NOTE]
+====
+AutoYaST is able to dump a profile from an existing installation.
+This can be used as template, but needs some adaptions to be usable from {productname}:
+====
+
+
+* Change the [literal]``add-on`` section and add {productname} URLs.
+These URLs must have a special format:
++
+
+----
+http://$redhat_management_server/ks/dist/child/<channel-label>/<distribution-label>
+----
+
+* Replace [literal]``<channel-label>`` and [literal]``<distribution-label>`` with the correct labels.
+It often makes sense to use a variable for `distribution-label`.
+Ensure that the distribution label corresponds to the Autoinstallable Distribution you have selected.
+You can only configure "child channels" here. These channels must be children of the base channel you selected in the distribution you use for this profile.
++
+[NOTE]
+====
+You do not need to specify a base channel.
+You defined the base channel in the distribution already.
+====
+
+* Register the system after it is installed.
+For this step we provide script snippets that can be used.
++
+For Salt managed clients, use the `spacewalk/sle_register_salt` snippet:
++
+----
+<scripts>
+  <init-scripts config:type="list">
+    $SNIPPET('spacewalk/sle_register_salt')
+  </init-scripts>
+</scripts>
+----
++
+For traditional clients, use the `spacewalk/sles_register_script` snippet:
++
+----
+<scripts>
+  <init-scripts config:type="list">
+    $SNIPPET('spacewalk/sles_register_script')
+  </init-scripts>
+</scripts>
+----
++
+[IMPORTANT]
+====
+When registering with Salt, the Salt key must be accepted on the {productname} Server before the autoinstallation procedure could be finished.
+====
+
+For more information about autoinstallation profiles, see xref:reference:systems/autoinst-profiles.adoc[].
+
+
+
+==== Variables and snippets
+
+The final state of a profile is composed when a client requests it.
+This allows you using variables in profiles.
+In [guimenu]``Profiles`` you can find the [guimenu]``Variables`` where you can define variables.
+
+Often used variables are:
+
+redhat_management_server::
+the server where you register to (automatically set).
+org::
+the organization ID where this profile is created (automatically set).
+registration_key::
+the key used in the registration snippets.
+When you specify it, you can set the activation key to be used to register the system.
+dont_register::
+if specified the registration will be skipped.
+allow_config_actions::
+if set to ``1``, it will allow traditional configuration management (traditional only).
+allow_remote_commands::
+if set to ``1``, it will allow traditional remote command execution (traditional only).
+dont_disable_automatic_onlineupdate::
+if set, the automatic online update will stay enabled (SUSE OSes only).
+dont_disable_local_repos::
+if set, local repositories will stay active (not recommended).
```

Comment:
The descriptions need capital letters, if you are giving them fullstops ;)

Reply:
Fair proposal ;) My excuse: the fullstops were already provided before I started editing (IIRC).